exporter/infra/exporter.sh

```diff
@@ -26,15 +51,15 @@ CLUSTER_API=$($OC cluster-info | \
 )
 # Check cluster API connection
 [[ $? -ne 0 ]] && { echo "Not connected to a cluster. Exiting..."; exit; }
-CLUSTER_ID="$($OC get clusterversion -o jsonpath='{.items[].spec.clusterID}{"\n"}')"
```

Comment:
Please always keep the original CLUSTER_ID as it is
Instead, we can rename variable associated to "-i" option to something like $USER_CLUSTER_ID and use it as the file name but not as the CLUSTER_ID value in the .env file

Reply:
Did it